### PR TITLE
Remove jax.jit from gradient calculations

### PR DESCRIPTION
**Pull request by Jashcraf**
See issue #137 , It looks like the extra jit around the gradient of a jit-compiled function just slows things down. So this just removes jax.jit from the jax_model_grads

### CODE REVIEW

**Comment by mihir-r-kondapalli** [comment]
LGTM
